.fossa.yml

```diff
@@ -0,0 +1,15 @@
+# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
+# Visit https://fossa.io to learn more
+version: 1
+cli:
+  server: https://app.fossa.io
+  fetcher: custom
+  project: https://github.com/projectcalico/app-policy.git
+analyze:
+  modules:
+  - name: github.com/projectcalico/app-policy
+    type: go
+    target: github.com/projectcalico/app-policy
+    path: /go/src/github.com/projectcalico/app-policy
+    options:
+      allow-unresolved: true
```

Comment:
Worth checking this difference to closed source?

Reply:
I don't see a difference from closed source here?